### PR TITLE
New Getting Started page

### PR DESCRIPTION
**Pull request by jospoortvliet**
New getting started page. Thanks to @tomneedham for starting it, @camilasan for the help and @jancborchardt @MorrisJobke for the feedback!

This is making #369 ready for merging.

@kelleybrooks @karlitschek would love to hear what you think. Screen shots below.
sshot0
Download button:
sshot1
sshot2
new 'cloud' page:
sshot3
Client button(s):
sshot4
sshot5
Testing button:
sshot6

Everything else redirects to other pages or gives a direct download.

What do we think?

### CODE REVIEW

**Comment by karlitschek** [comment]
great. As discussed. i love it. :+1: 
The only thing we could add is the option to buy a device that offers ownCloud hosting. Like linking to qnap and synology and other who offer owncloud apps that can run on top of then.
This requires some investigation what exactly is where possible and available and what the links are.
No sure who could do that.

**Comment by jospoortvliet** [comment]
@karlitschek I actually thought about that, but for what you mention - it requires some more looking into. Something we can add later, I would say...

**Comment by karlitschek** [comment]
ok. agreed. The rest is great!!

**Comment by kelleybrooks** [comment]
Was just thinking yesterday how much the .org site has evolved.  This is really great stuff.  My suggestions are tiny...
-on Web Installer, a word other than novice would be good.  Some people just choose not to host.  It may be good to link to our providers page as that is constantly growing and I don't believe we link to it directly from nav/footer
-on desktop clients, we use the British spelling on "synchronise".  I know that much of the English on the site is British English whereas .com is US English.  You're call.  Just thought I'd highlight.
If I saw anything else, it's since escaped my brain.  These are a huge step in clarity....

**Comment by jospoortvliet** [comment]
addressed the comments by Kelley in the above commit as well. Keeping British English where it is but removed the 'novice'.

**Comment by jospoortvliet** [comment]
@jancborchardt I'd really like to hear your thoughts before I merge to staging and get it live... I have not yet made the changes needed to replace the current install/download pages.

**Comment by jancborchardt** [comment]
It looks really nice! This also then replaces the /install and /sync-client pages, right? (Cause otherwise it would be a massive duplication of content it seems)

Just one thing: The modal should be shifted a bit down, as it currently weirdly overlays the header.

**Comment by jospoortvliet** [comment]
@jancborchardt yeah, it should replace them.

Let me see if I can figure out how to move those modal dialogs down a bit, then I'll consider this a :+1: and will merge it to staging...

**Comment by jospoortvliet** [comment]
You can see it on staging and play with it yourself: https://staging.owncloud.org/getstarted/

if no more problems found I will try and figure out how to get it to replace the install page... probably needs help from @tomneedham

File: start-instructions.php [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+                          <li><a href="#tab-web" title="Best for shared hosts" role="tab" data-toggle="tab"><i class="icon-code"></i> Web Installer<br><small>For shared hosts</small></a></li>
+                          <li><a href="#tab-packages" title="Provides automated updates" role="tab" data-toggle="tab"><i class="icon-linux"></i> Packages<br><small>For auto updates</small></a></li>
+                          <li><a href="#tab-cloud" title="Provides automated updates" role="tab" data-toggle="tab"><i class="icon-cloud"></i> Images<br><small>For cloud deployment</small></a></li>
+                      </ul>
+                  </div>
+                  <br>
+                  <div class="overlay-header row">
+                    <p>Latest stable version:  <span class="label label-success"><?php echo $VERSIONS_SERVER_FULL_STABLE; ?></span> (<a href="/changelog"><small>Changelog</small></a>)</br>
+                    <p>ownCloud can be installed on Windows and Linux operating systems, and is available via packages, sources, images or a one file php installer.</p>
```
Comment (MorrisJobke) [inline start-instructions.php]: @jospoortvliet I would remove the windows ;) We don't support it anymore

**Comment by MorrisJobke** [comment]
Clicking on "mobile clients" in step two: Wrong tab is selected (it should be mobile clients but is desktop client) - the content is correct.

**Comment by jospoortvliet** [comment]
@MorrisJobke can't figure out how to fix the mobile clients thing. It should be done by the script code on the bottom of the page but it doesn't work and I dunno why :(

About the Windows thing - not yet, it works for now and we stop support for 8.1, right?

**Comment by MorrisJobke** [comment]
No it was dropped for 8.0 AFAIK

**Comment by karlitschek** [comment]
I really like it. :+1: from my side.
I think we should still keep the zip download option. But remove the mentioning of "windows"

**Comment by MorrisJobke** [comment]
> I think we should still keep the zip download option. But remove the mentioning of "windows"

Yes. Sounds good :)

**Comment by LukasReschke** [comment]
- [ ] The OBS PGP key is missing on the packages site
- [ ] "Looking for older versions?" is not really in the middle

**Comment by jospoortvliet** [comment]
- [x] remove mentions of "windows"
- [ ] wrong tab is selected (content is correct) when clicking on mobile apps link

**Comment by jospoortvliet** [comment]
You guys are pushing me out of my comfort zone. I just managed to fix the tabs - that's, technically, javascript stuff. And I only modified my first CSS a month ago. Curse you!

**Comment by jospoortvliet** [comment]
@LukasReschke see #469 - all changes made. Push this to staging too, then?

**Pull request merged**